standalone/testkit/src/mill/standalone/StandaloneTester.scala

```diff
@@ -0,0 +1,104 @@
+package mill.standalone
+
+import mill.constants.DaemonFiles.processId
+import mill.constants.OutFiles.{millDaemon, millNoDaemon, out}
+import os.{Path, ProcessInput, ProcessOutput, Shellable}
+
+class StandaloneTester(
```

Comment:
Why does this need to be different from normal IntegrationTester?

Reply:
It doesn't have the same requirements as `IntegrationTester`. The workspace creation and initialization happens outside the tester.
Given that this PR proposal will take some time, I chose to isolate this to avoid any conflicts when re-basing.

Reply:
`IntegrationTester` clears the workspace and copies resources. I suppose this can be circumvented with an empty resource folder.

I am not opposed to the idea of sharing existing code. I just kept all new code isolated initially. I will merge the testkit classes and move the tests to `integration/manual`.